### PR TITLE
Ignore component accessibilty attrs failing test

### PR DESCRIPTION
**Pull request by cherifGsoul**
The accessibility attributes should be ignored in component viewModel

### CODE REVIEW

**Comment by justinbmeyer** [comment]
@cherifGsoul Thanks for submitting this.  I think if the viewModel is expecting a role attr, role should be bindable.  We might need a way to make things like id / class / role / etc work.


There's a few problems with the test (I'll comment on those in a second).

File: component/component_test.js [hunk 1]
```diff
@@ -1556,4 +1556,29 @@ steal("can", "can/map/define", "can/component", "can/view/stache" ,"can/route",
...
+		var frag=can.stache('<x-app></x-app>')({});
+		
+		var el = frag.childNodes[0];
+		var viewModel = can.viewModel(el);
+		
+		// element must be inserted, otherwise attributes event will not be fired
+		can.append(can.$("#qunit-fixture"),frag);
+		
+		// update the class
+		can.$(el).attr("role","foo");
```
Comment (justinbmeyer) [inline component/component_test.js]: This won't work outside jQuery.  You probably want to use can.attr.set.

**Comment by cherifGsoul** [comment]
The problem is if the viewModel will be in the can.route.map the attribute can be displayed in the url for example: /#!some-path&role=button

**Comment by cherifGsoul** [comment]
@justinbmeyer here is a fiddle http://jsfiddle.net/cherif_b/88drbcxx/ that demonstrate the issue
sorry if im late I was busy with work :)

**Comment by justinbmeyer** [comment]
this is closed for 3.0.

**Pull request closed without merging**